addon/components/time-series-chart.js

```diff
@@ -240,7 +282,7 @@ const TimeSeriesChartComponent = ChartComponent.extend(LegendMixin,
   }),
 
   graphicHeight: Ember.computed('height', 'legendHeight', 'legendChartPadding', function() {
-    return this.get('height') - this.get('legendHeight') - this.get('legendChartPadding');
+    return this.get('height') - this.get('legendHeight') - this.get('legendChartPadding') - (this.get('MarginBottom') || 0 );
   }),
```

Comment:
- Typo: s/MarginBottom/marginBottom/g
- Can we initialize this.marginBottom to avoid the or case?
- The function body should probably be broken up into multiple lines anyway.
- We need to add a dependency on this.marginProperty to this computed property.